code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -27,12 +27,11 @@
 		var/list/meancritters = list() // list of possible hostile mobs
 		var/list/nicecritters = list() // and possible friendly mobs
 		for(var/T in typesof(/mob/living/simple_animal))

```

Comment:
See my comment below, you can't do the typecast in the for statement, but CAN do it in the for block.